about.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>SciPy India</title>
+    <link rel="icon" href="assets/SciPy-India-logo.png" type="image/png">
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <header>
+      <special-header></special-header>
+    </header>
+
+    <main>
+      <section id="about">
+        <h2>About Us</h2>
+        <p>SciPy India is a community and an annual conference centred around the Python programming language for education and scientific computing topics In India</p>
+      
+        <ul class="blog-list">
+          <li><a href="blog1.html">About SciPy India</a></li>
+          <li><a href="blog2.html">Highlights from Community Call #1</a></li>
+          <!-- <li><a href="blogpost3.html">Community Initiatives</a></li> -->
+        </ul>
+      </section>
+      
+      </main>
+      
+      <footer>
+        <special-footer></special-footer>
+      </footer>
+
+      <script src="./headerFooterManager.js"></script>
+  </body>
+</html>
```

Comment:
```suggestion
</html>

```

Nit: no newline at end-of-file